visualization/R/add_icon.R

```diff
@@ -0,0 +1,18 @@
+##' @name add_icon
+##' @param x x-coordinate of logo
+##' @param y y-coordinate of logo
+##' @export 
+##' @author Mike Dietze
+##' 
+add_icon <- function(id=NULL,x=0,y=0){
+  require(png)
+  require(grid)
```

Comment:
Why `require` instead of `library`? This is not usually a good idea.

E.g., if the `png` package isn't available, a warning will be generated, but the code will keep running. But _this doesn't make any sense_, because it won't work without that package.